hazelcast/src/main/java/com/hazelcast/internal/server/tcp/TcpServerAcceptor.java

```diff
@@ -288,7 +289,12 @@ private void newConnection(final EndpointQualifier qualifier, SocketChannel sock
             if (logger.isFineEnabled()) {
                 logger.fine("Accepting socket connection from " + channel.socket().getRemoteSocketAddress());
             }
-
+            serverContext.getAuditLogService()
+                .eventBuilder(AuditlogTypeIds.NETWORK_CONNECT)
+                .message("New connection accepted.")
```

Comment:
I guess we should make clear in the documentation that these invocations are done in performance sensitive places so the logger appender should not do anything too heavy.

Reply:
I'll add a note to the `AuditloService` javadoc.